src/widgets/css_provider.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright 2013-2015, The Rust-GNOME Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+// TODO: feature = ?
+
+use std::fmt::{self, Display, Formatter};
+use ffi::{self, GtkCssProvider};
+use glib::translate::{ToGlibPtr, from_glib_full};
+use glib::{self, GlibContainer};
+
+#[repr(C)]
+pub struct CssProvider {
+    pointer: *mut GtkCssProvider
+}
+
+impl ::StyleProviderTrait for CssProvider {}
+
+impl ::FFIStyleProvider for CssProvider {
+    fn unwrap_provider(&self) -> *mut ffi::GtkStyleProvider {
+        unsafe { ffi::cast_GtkStyleProvider(self.pointer as *mut ffi::GObject) }
+    }
+}
+
+impl CssProvider {
+    pub fn new() -> Self {
+        unsafe { CssProvider { pointer: ffi::gtk_css_provider_new() } }
+    }
+
+    pub fn get_default() -> Self {
+        unsafe { CssProvider { pointer: ffi::gtk_css_provider_get_default() } }
+    }
+
+    pub fn get_named(name: &str, variant: &str) -> Self {
+        unsafe {
+            CssProvider { pointer: ffi::gtk_css_provider_get_named(name.to_glib_none().0,
+                                                                   variant.to_glib_none().0) }
+        }
+    }
+
+    pub fn load_from_path(path: &str) -> Result<CssProvider, glib::Error> {
+        unsafe {
+            let pointer = ffi::gtk_css_provider_new();
+            let mut error: *mut glib::ffi::GError = ::std::mem::zeroed();
```

Comment:
Ah, I was looking for how to do that. I saw `ptr::null()`, but missed `ptr::null_mut()`.